app/dash_app.py

```diff
@@ -1,383 +1,397 @@
+                        _read_coherence(_coherence_csv(INITIAL_TARGET))),
+                    style={
+                        'height': '40%',
+                    }),
+            ],
+            style={
+                'height': 800,
+                'marginLeft': LR_MARGIN_PX,
+                'marginRight': LR_MARGIN_PX,
+            }
         ),
 
-    # html.Div(),
-
-    # html.Div(style={'width': '5%', 'display': 'inline-block'}),
-    # html.Div(
-    #     dl.Map([dl.TileLayer(),
-    #             dl.LayersControl(
-    #                 dl.BaseLayer(
-    #                         dl.TileLayer(
-    #                             # Selection of Basemaps
-    #                             url='https://basemap.nationalmap.gov/arcgis/rest/services/USGSTopo/MapServer/tile/{z}/{y}/{x}',
-    #                             attribution='Tiles courtesy of the <a href="https://usgs.gov/">U.S. Geological Survey</a>'
-    #                         ),
-    #                         name="USGS Topo",
-    #                         checked=True
-    #                         ),
-    #                     ),
-    #             dl.WMSTileLayer(id='rmlimap',
-    #                             url=f"{config.get('geoserver', 'geoserverEndpoint')}/Meager_5M3/wms?",
-    #                             layers="cite:20210114_HH.rmli.geo.db",
-    #                             format="image/png",
-    #                             transparent=True,
-    #                             opacity=1.0),
-    #                 ],
-    #            id='leafletMapRMLI',
-    #            center=[50.64, -123.6],
-    #            zoom=12
-    #            ),
-    #     style={'width': '90%', 'height': '550px', 'display': 'inline-block'}
-    #     ),
-    # html.Div([
-    #     dcc.Slider(min=0,
-    #                max=len(dfCohFull.dropna()['Reference Date'].unique()),
-    #                step=1,
-    #                marks={i: dfCohFull.dropna()['Reference Date'].unique()[i] for i in range(0, len(dfCohFull.dropna()['Reference Date'].unique()), 10)},
-    #                value=0,
-    #                id='my-slider'
-    #                ),
-    #     html.Div(id='slider-output-container')
-    #         ]),
-    # html.Div(),
-    ])
-
-
-# Update interferogram selection displayed on leaflet map
-@app.callback(
-    Output(component_id="map", component_property="layers"),
-    Input(component_id="graph", component_property="clickData"))
-def update_datepair(clickData):
-    if not clickData:
-        return 'cite:20210717_HH_20210903_HH.adf.wrp.geo'
-    dates = '{}_HH_{}_HH'.format(json.dumps(clickData['points'][0]['x'],
-                                            indent=2),
-                                 json.dumps(clickData['points'][0]['y'],
-                                            indent=2))
-    dates = dates.replace('-', '').replace('"', '')
-    print(dates)
-    return 'cite:{}.adf.wrp.geo'.format(dates)
+        # html.Div(
```

Comment:
Let's go ahead and remove this commented code. It is in the git history if we ever need it